packages/larva-patterns/modules/content-showcase-grid/content-showcase-grid.twig

```diff
@@ -0,0 +1,7 @@
+<div class="content-showcase-grid larva // {{ content_showcase_classes }}">
+	<div class="{{ content_showcase_items_classes }}">
+		{% for item in content_showcase_items %}
+			{% include "@larva/modules/content-showcase/content-showcase.twig" with item %}
```

Comment:
We should not nest modules in modules. Is there an object that can be used here? Perhaps o-card?

Reply:
You could also include components instead of creating a new object, if none of the objects work here.

Reply:
@laras126 This has been fixed in the updated template. Using o-card now